### PR TITLE
migrate code from googleapis/python-bigquery-datatransfer

### PR DESCRIPTION
**Pull request by dizcology**
- BigQuery: Add Data Transfer Service quickstart. [(#1295)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1295)
- Auto-update dependencies. [(#1377)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1377)
- bigquery_datatransfer -> bigquerydatatransfer
- Auto-update dependencies. [(#1846)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1846)
- Auto-update dependencies. [(#1980)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/1980)
- Adds updates including compute [(#2436)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2436)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v1 [(#3208)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/3208)
- Simplify noxfile setup. [(#2806)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/2806)
- Replace GCLOUD_PROJECT with GOOGLE_CLOUD_PROJECT. [(#4022)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4022)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v1.1.0 [(#4215)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4215)
- chore(deps): update dependency pytest to v5.4.3 [(#4279)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4279)
- chore(deps): update dependency mock to v4 [(#4287)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4287)
- Update dependency pytest to v6 [(#4390)](https://github.com/GoogleCloudPlatform/python-docs-samples/issues/4390)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v1.1.1 (#55)
- chore: transition the library to microgenerator (#56)
- chore: Adjust gapic namespace to google.cloud.bigquery_datatransfer (#61)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v2 (#59)
- docs: add sample for dataset copy (#76)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3 (#81)
- docs: remove out-of-date sample from README (#80)
- docs: add scheduled query samples (#83)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.0.1 (#111)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.1.0 (#120)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.1.1 (#127)
- chore(deps): update dependency pytest to v6.2.3 (#137)
- chore(deps): update dependency mock to v4.0.3 (#135)
- chore(deps): update dependency google-cloud-bigquery to v2.16.0 (#134)
- chore(deps): update dependency pytest to v6.2.4 (#147)
- chore(deps): update dependency google-cloud-bigquery to v2.20.0 (#154)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.2.0 (#168)
- docs: add sample to include run notification (#173)
- chore: use gapic-generator-python 0.50.4 (#174)
- chore(deps): update dependency google-cloud-pubsub to v2.6.1 (#175)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.3.0 (#176)
- feat: add Samples section to CONTRIBUTING.rst (#181)
- chore(deps): update dependency google-cloud-bigquery to v2.22.1 (#177)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.3.1 (#188)
- chore(deps): update dependency google-cloud-bigquery to v2.23.0 (#189)
- chore(deps): update dependency google-cloud-bigquery to v2.23.1 (#191)
- chore(deps): update dependency google-cloud-pubsub to v2.7.0 (#190)
- chore(deps): update dependency google-cloud-bigquery to v2.23.2 (#193)
- chore(deps): update dependency google-cloud-bigquery to v2.24.0 (#195)
- chore(deps): update dependency google-cloud-pubsub to v2.7.1 (#204)
- chore(deps): update dependency google-cloud-bigquery to v2.25.1 (#203)
- chore(deps): update dependency pytest to v6.2.5 (#210)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.3.2 (#212)
- chore(deps): update dependency google-cloud-bigquery to v2.25.2 (#213)
- chore(deps): update dependency google-cloud-bigquery to v2.26.0 (#216)
- chore(deps): update dependency google-cloud-pubsub to v2.8.0 (#218)
- chore(deps): update all dependencies (#224)
- chore(deps): update dependency google-cloud-bigquery to v2.27.1 (#225)
- chore(deps): update dependency google-cloud-bigquery to v2.28.0 (#229)
- chore(deps): update all dependencies (#231)
- chore(deps): update dependency google-cloud-bigquery to v2.29.0 (#240)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.4.1 (#243)
- chore(deps): update dependency google-cloud-bigquery to v2.30.0 (#245)
- chore(deps): update dependency google-cloud-bigquery to v2.30.1 (#246)
- chore(deps): update dependency google-cloud-pubsub to v2.9.0 (#249)
- chore(deps): update dependency google-cloud-bigquery to v2.31.0 (#256)
- chore(deps): update dependency google-cloud-bigquery to v2.32.0 (#269)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.5.0 (#272)
- chore(deps): update dependency pytest to v7 (#282)
- chore(deps): update dependency pytest to v7.0.1 (#284)
- chore(deps): update all dependencies (#285)
- chore(deps): update all dependencies (#294)
- chore(deps): update dependency google-cloud-pubsub to v2.10.0 (#299)
- chore(deps): update all dependencies (#301)
- chore(deps): update dependency google-cloud-pubsub to v2.11.0 (#302)
- chore(deps): update dependency pytest to v7.1.0 (#305)
- chore(deps): update dependency pytest to v7.1.1 (#306)
- chore(python): use black==22.3.0 (#310)
- chore(deps): update dependency google-cloud-bigquery to v2.34.3 (#311)
- chore(deps): update dependency google-cloud-bigquery to v3 (#314)
- chore(deps): update dependency google-cloud-pubsub to v2.12.0 (#320)
- chore(python): add nox session to sort python imports (#326)
- chore(deps): update dependency pytest to v7.1.2 (#329)
- chore(deps): update dependency google-cloud-bigquery to v3.1.0 (#334)
- chore(deps): update dependency google-cloud-pubsub to v2.12.1 (#335)
- chore(deps): update all dependencies (#354)
- chore(deps): update all dependencies (#370)
- chore(deps): update all dependencies (#372)
- chore(deps): update dependency google-cloud-pubsub to v2.13.5 (#374)
- chore(deps): update dependency google-cloud-pubsub to v2.13.6 (#377)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.7.1 (#379)
- chore(deps): update dependency google-cloud-bigquery to v3.3.2 (#380)
- chore(deps): update dependency pytest to v7.1.3 (#389)
- chore(deps): update all dependencies (#395)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.7.2 (#399)
- chore(deps): update all dependencies (#402)
- chore(deps): update dependency pytest to v7.2.0 (#403)
- chore(deps): update all dependencies (#405)
- chore(deps): update dependency google-cloud-bigquery to v3.4.1 (#413)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.8.0 (#414)
- chore(deps): update dependency mock to v5 (#415)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.9.0 (#418)
- chore(deps): update dependency mock to v5.0.1 (#420)
- chore(deps): update dependency google-cloud-pubsub to v2.13.12 (#423)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.10.0 (#424)
- chore(deps): update dependency pytest to v7.2.1 (#425)
- chore(deps): update dependency google-cloud-bigquery to v3.4.2 (#426)
- chore(deps): update dependency google-cloud-pubsub to v2.14.0 (#427)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.10.1 (#430)
- chore(deps): update dependency google-cloud-bigquery to v3.5.0 (#434)
- chore(deps): update dependency google-cloud-pubsub to v2.14.1 (#438)
- chore(deps): update all dependencies (#442)
- chore(deps): update dependency google-cloud-bigquery-datatransfer to v3.11.0 (#446)
- chore(deps): update dependency pytest to v7.2.2 (#447)
- fix: replaces deprecated snippet (#448)
- docs: removes deprecated sample (#451)
- chore(deps): update all dependencies (#452)
- chore(deps): update all dependencies (#457)
- chore(deps): update dependency google-cloud-bigquery to v3.10.0 (#459)
- chore(deps): update dependency google-cloud-pubsub to v2.17.1 (#460)
- chore(deps): update dependency google-cloud-bigquery to v3.11.0 (#463)
- chore(deps): update all dependencies (#466)

## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [ ] Please **merge** this PR for me once it is approved

### CODE REVIEW

File: .github/CODEOWNERS
Comment (rsamborski) [inline .github/CODEOWNERS:74]: Please add `/bigquery-datatrasfer/**/*` to CODEOWNERs file.

**Review by rsamborski — APPROVED** [review]
Approved after addressing CODEOWNERs comment. The PR will require an admin merge due to CLA exceptions.

File: .github/CODEOWNERS
Comment (dizcology) [reply .github/CODEOWNERS:74]: Done.  Thanks!

**Comment by dizcology** [comment]
Updated the CODEOWNERS change to address the comment.

Next steps:
From @leahecole: "reminder for whoever merges to do "Rebase and merge" from the dropdown to preserve history!"
From @rsamborski: "The PR will require an admin merge due to CLA exceptions."

**Review by leahecole — APPROVED** [review]

**Comment by leahecole** [comment]
CLA failure is from a xoogler and is ok

**Pull request merged**
